RecDP/pyrecdp/LLM/TextPipeline.py

```diff
@@ -338,6 +341,8 @@ def execute(self):
         op_chain = []
 
         if engine_name == 'ray':
+            if ray.is_initialized():
+                ray.shutdown()
```

Comment:
@chaojun-zhang , this is not acceptable, you may directly close a ray for customer task , for example in llm-ray integration, they will init ray